doc-Infrastructure_Migration_Solution_Guide/cfme/modules/ref_Target_prerequisites.adoc

```diff
@@ -5,7 +5,28 @@
 [id="Target_prerequisites_{context}"]
 = Prerequisites
 
-The following prerequisites apply:
+Ensure that the following prerequisites are met:
 
-* BIOS settings of physical hosts adjusted for optimal performance (rather than power-saving), according to the vendor's recommendations
-* C1E halt state disabled, if applicable
+. Set the BIOS settings of physical hosts for optimal performance (rather than power-saving), according to the vendor's recommendations.
+. Disable C1E halt state, if applicable.
+ifdef::rhv[]
+. Enable the following ports in the conversion host network:
+* 22 - SSH
+* 443 - CloudForms, Red Hat Virtualization Manager, and VDDK
+* 902 - CloudForms to VMware
+* 5480 - Conversion hosts to vCenter
++
+For details, see https://access.redhat.com/articles/417343[Ports used by Red Hat CloudForms Management Engine 5.1 and above].
```

Comment:
Per the final comment on this page, the list in now maintained in the CloudForms Appliance Hardening Guide.  I would recommend updating the link to point to: https://github.com/ManageIQ/manageiq_docs/blob/master/doc-Appliance_Hardening_Guide/topics/Firewall.adoc

Reply:
Thanks@